app/monitoringapi_internal_test.go

```diff
@@ -113,7 +113,7 @@ func TestStartChecker(t *testing.T) {
 			if tt.err != nil {
 				require.Eventually(t, func() bool {
 					err = readyErrFunc()
-					if err != nil {
+					if !errors.Is(err, errReadyUnInit) {
```

Comment:
it was failing because we are checking for err != nil while we have `errReadyUnInit` as default initially.

Reply:
`...UnInit` doesn't make sense

Reply:
was suggested by you 🙃